centreon/packages/ui/src/Panel/Section/index.tsx

```diff
@@ -57,8 +60,10 @@ interface Section {
 }
 
 interface SectionPanelProps {
+  currentWidth?: (width: number) => void;
```

Comment:
what is currentWidth in relation to mainPanelWidth? it's not very clear to me. Is it the total panel width? in which case, it should simply be named width